modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessagingEngine.java

```diff
@@ -136,20 +133,22 @@ public void initialise(MessageStore messageStore, MessageExpiryManager messageEx
      * @return AndesMessagePart
      * @throws AndesException
      */
+    // TODO
     public AndesMessagePart getMessageContentChunk(long messageID, int offsetInMessage) throws AndesException {
         return messageStore.getContent(messageID, offsetInMessage);
     }
 
     /**
      * Read content for given message metadata list
      *
-     * @param messageIdList message id list for the content to be retrieved
+     * @param longArrayList message id list for the content to be retrieved
      * @return <code>Map<Long, List<AndesMessagePart>></code> Message id and its corresponding message part list
      * @throws AndesException
      */
-    public LongObjectHashMap<List<AndesMessagePart>> getContent(LongArrayList messageIdList) throws AndesException {
-        return messageStore.getContent(messageIdList);
-
+    public HashMap<Long, List<AndesMessagePart>> getContent(String queueName, LongArrayList longArrayList)
+            throws AndesException {
+        HashMap<Long, List<AndesMessagePart>> content = messageStore.getContent(queueName, longArrayList);
```

Comment:
Removed local variable.
Now it is return messageStore.getContent(queueName, messageIDList);